tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "allowSyntheticDefaultImports": true,
     "sourceMap": true,
     "baseUrl": ".",
-    "types": ["webpack-env", "jest"],
+    "types": ["webpack-env", "jest", "googlemaps"],
```

Comment:
Lol, im still trying to figure out why the TS `types` config in my Leaflet plugin doesnt work.... grrr.

Reply:
Hm... yea, maybe have a look at the actual type definition and see if its actually declaring a global namespace or not?

Reply:
Good idea!